js/data/symbol_bucket.js

```diff
@@ -379,10 +465,8 @@ SymbolBucket.prototype.updateFont = function(stacks) {
 };
 
 SymbolBucket.prototype.addToDebugBuffers = function(collisionTile) {
-
     this.elementGroups.collisionBox = new ElementGroups(this.buffers.collisionBoxVertex);
-    this.elementGroups.collisionBox.makeRoomFor(0);
-    var buffer = this.buffers.collisionBoxVertex;
+    this.makeRoomFor('collisionBox', 0);
```

Comment:
We don't create the first element group (i.e. `elementGroups.current === null`) until `makeRoomFor` is called. I agree this is awkward.